src/main/java/frc/robot/Robot.java

```diff
@@ -25,23 +29,24 @@ public void robotInit() {
 
   @Override
   public void configureButtonBindings() {
-    DoubleSupplier trigger = controller::getTriggers;
+    driveController.start().whenPressed(drive.resetCmd());
+
+    DoubleSupplier trigger = driveController::getTriggers;
 
     // Move with variable speed from triggers
-    controller.x().whileHeld(parallel("Move", leftClimber.move(trigger), rightClimber.move(trigger)));
+    driveController.x().whileHeld(parallel("Move", leftClimber.move(trigger), rightClimber.move(trigger)));
 
     // Button bindings for regular climbing
-    controller.a().whileHeld(parallel("Extend", leftClimber.extend(), rightClimber.extend()));
-    controller.b().whileHeld(parallel("Retract", leftClimber.retract(), rightClimber.retract()));
+    driveController.a().whileHeld(parallel("Extend", leftClimber.extend(), rightClimber.extend()));
+    driveController.b().whileHeld(parallel("Retract", leftClimber.retract(), rightClimber.retract()));
 
     // Button bindings for ignoring limit switches
-    controller.getPovButton(POVDirection.UP)
+    driveController.getPovButton(POVDirection.UP)
         .whileHeld(parallel("Extend Ignore Limit", leftClimber.extendIgnoreLimit(), rightClimber.extendIgnoreLimit()));
-    controller.getPovButton(POVDirection.DOWN).whileHeld(
+    driveController.getPovButton(POVDirection.DOWN).whileHeld(
         parallel("Retract Ignore Limit", leftClimber.retractIgnoreLimit(), rightClimber.retractIgnoreLimit()));
 
-    controller.start().whenPressed(parallel("Stop", leftClimber.stop(), rightClimber.stop()));
-
+    driveController.start().whenPressed(parallel("Stop", leftClimber.stop(), rightClimber.stop()));
```

Comment:
Just a warning, this is also mapped to `drive.resetCmd()` up above.

Reply:
Yeah probably good to remap this to `select()` instead (and maybe do the same for all commands?)